data/operators/amenity/school.json

```
@@ -18726,6 +18726,16 @@
         "operator": "Makilala West District of North Cotabato Schools Division, DepEd"
       }
     },
+    {
+      "displayName": "Maktab Rendah Sains MARA",
+      "locationSet": {"include": ["my"]},
+      "tags": {
+        "amenity": "school",
+        "operator": "Majlis Amanah Rakyat",
+        "operator:short": "MARA",
+        "operator:wikidata": "Q4263028"
```

I noticed that you deliberately emphasized the schools created by Majlis Amanah Rakyat (MARA, a Malaysian government agency).

If you wish, you can add the following `operator:type` tags:

```suggestion
        "operator:wikidata": "Q4263028",
        "operator:type": "government"
```

~~(Of course, this is just a suggestion that came to me after seeing your introductory description. It simply means that I think it might be more detailed. Whether or not to adopt this suggestion depends on you.)~~